cmd/vc-rest/startcmd/start.go

```diff
@@ -11,6 +11,8 @@ import (
 	"crypto/subtle"
 	"crypto/tls"
 	"fmt"
+	"github.com/trustbloc/vcs/pkg/storage/mongodb/cslstore"
+	"github.com/trustbloc/vcs/pkg/storage/mongodb/vcstore"
```

Comment:
move to the last section of import block

Reply:
done.